tests_integration/__snapshots__/test_git.ambr

```diff
@@ -3,7 +3,16 @@
   list([
     dict({
       'Error': None,
-      'File': 'snowcli_testing_repo/branches/main/tests_integration/test_data/projects/stage_execute/script_template.sql',
+      'File': '@SNOWCLI_TESTING_REPO/branches/main/tests_integration/test_data/projects/stage_execute/script_template.sql',
+      'Status': 'SUCCESS',
+    }),
+  ])
+# ---
+# name: test_execute.1
+  list([
+    dict({
+      'Error': None,
+      'File': '@db_f1ae203df88c4b80bf3fc31e5431baca.public.SNOWCLI_TESTING_REPO/branches/main/tests_integration/test_data/projects/stage_execute/script_template.sql',
```

Comment:
This doesn't work because db is always different on each test run. We should consider other way to check than snapshot